brd-ios/breadwallet/src/Redesign/Views/Core/FEPopupView.swift

```diff
@@ -104,7 +104,7 @@ class FEPopupView: FEView<PopupConfiguration, PopupViewModel> {
         closeButtonContainer.addSubview(closeButton)
         closeButton.snp.makeConstraints { make in
             make.trailing.top.equalToSuperview().inset(Margins.huge.rawValue)
-            make.width.height.equalTo(ViewSizes.extraSmall.rawValue)
+            make.width.height.equalTo(Margins.medium.rawValue)
```

Comment:
check closeButton margins in the variable. please, make sure the button tappable area is large. changing sizes can affect that

Reply:
yes, the tappable area for the button is in the closeButtonContainer not in the closeButton, it's large